pmlc/dialect/pxa/transforms/passes.td

```diff
@@ -8,6 +8,11 @@ def AutoTileExample : FunctionPass<"pxa-autotile-example"> {
   let constructor = "pmlc::dialect::pxa::createAutoTileExamplePass()";
 }
 
+def TileAccumulate : FunctionPass<"pxa-tile-accumulate"> {
+  let summary = "Tile accumulate";
```

Comment:
Could you add a slightly better description?

Reply:
Yes, I will add more descriptive words.